### PR TITLE
Add single namespace tier

### PR DESCRIPTION
**Pull request by rajivnathan**
Creates a new tier `baseonens` that only has 1 namespace (`-dev`)

Related PR: https://github.com/codeready-toolchain/toolchain-e2e/pull/499

### CODE REVIEW

**Review by alexeykazakov — COMMENTED** [review]
Apart from the name the PR looks good :)

**Review by MatousJobanek — COMMENTED** [review]
One correction plus - as it was mentioned by Alexey - rename it to `base1ns`

**Comment by rajivnathan** [comment]
Renamed to `base1ns` in 5082866eda62790ff385218ba4bc5baa65725fcf

File: deploy/templates/nstemplatetiers/base1ns/cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    name: for-${USERNAME}-sbo
+  spec:
+    quota:
+      hard:
+        count/servicebindings.binding.operators.coreos.com: "50"
+        count/servicebindings.servicebinding.io: "50"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
```
Comment (alexeykazakov) [inline deploy/templates/nstemplatetiers/base1ns/cluster.yaml:154]:
```suggestion
```

**Review by MatousJobanek — APPROVED** [review]
Looks good, just remove the RHOAS & SBO  ClusterResourceQuotas

**Review by alexeykazakov — APPROVED** [review]

**Pull request merged**